range.c

```diff
@@ -554,6 +553,33 @@ range_step(int argc, VALUE *argv, VALUE range)
     return range;
 }
 
+/*
+ *  call-seq:
+ *    %(n) {|element| ... } -> self
+ *    %(n)                  -> enumerator
+ *
+ *  Iterates over the elements of +self+.
+ *
+ *  With a block given and no argument,
+ *  calls the block each element of the range; returns +self+:
```

Comment:
The example shows the block is not called with each element of the range.  The example shows the block is called with every nth element in the range, as specified by the argument.  So this should be reworded.

Reply:
Done.